server/routes.js

```diff
@@ -548,3 +548,34 @@ exports.mountQuintypeAt = function(app, mountAt) {
     }
   });
 };
+
+/**
+ * ampRoutes uses quintype-amp library to handle amp pages
+ * currently, only story pages with default styling are supported
+ */
```

Comment:
please document parameters and such. Please see `proxyGetRequest` as an example

Reply:
Okay. Adding documentation